users/tests.py

```diff
@@ -1,3 +1,65 @@
+from rest_framework.test import APIClient
 from django.test import TestCase
 
-# Create your tests here.
+
+class UserRegistrationTestCase(TestCase):
+    def test_user_can_register(self):
+        client = APIClient()
+        response = client.post(
+            '/api/profiles/create/',
```

Comment:
You should generate the urls with using reverse method. `django.urls.reverse`. Urls may change in the future.